test_common/capybara/capybara_driver_helper.rb

```diff
@@ -1,7 +1,7 @@
 require 'capybara'
 require_relative '../configuration'
 Capybara.configure do |config|
-  driver = ENV.fetch('DRIVER', 'chromedriver').to_sym
+  driver = ENV.fetch('DRIVER', 'chromedriver').to_sym || :headless
```

Comment:
The || :headless will not achieve anything.  If DRIVER is not set in the environment it will default to 'chromedriver'.
If you want headless (I find chrome headless unreliable though) then use
DRIVER=chromedriver_headless
in your .env file